content/docs/next-release/deployment.md

```diff
@@ -171,6 +171,117 @@ docker run \
   jaegertracing/all-in-one:{{< currentVersion >}}
 ```
 
+#### Upgrade Badger v1 to v3
```

Comment:
We usually do not keep such detailed migration instructions in the core documentation - they are going to become irrelevant soon, but the docs are "forever". I would instead create an .md file in the badger dir in the main repo and link to that file from the changelog.

Reply:
True, I think it might be useful to have it here for this version, and remove for the next one, keeping the doc in the readme.